modules/cnf-creating-an-additional-sriov-network-with-vrf-plug-in.adoc

```diff
@@ -62,37 +65,38 @@ spec:
 $ oc create -f sriov-network-attachment.yaml
 ----
 
-.Verifying that the `NetworkAttachmentDefinition` CR is successfully created
+.Verification
```

Comment:
multi-step verifications i would probably modularize; i know we have a rule now that these should be one step--not sure how these longer ones will really translate in dita